Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -86,5 +87,7 @@ void GameListModel::RemoveGame(QString path)
 	beginRemoveRows(QModelIndex(), entry, entry);
 	m_entries.remove(path);
 	m_games.removeAt(entry);
+	for (int i = entry; i < m_games.size(); i++)
```

Comment:
What does this do in particular? It may be worth breaking this loop out into its own self-descriptive named function